### PR TITLE
Added long-awaited feature 'non-stop-node'

### PR DESCRIPTION
**Pull request by insoPL**

### CODE REVIEW

File: typingtraining [hunk 1]
```diff
@@ -11,18 +11,22 @@ from lib.tracker import Tracker
...
     stdscr.addstr('To proceed press "BACKSPACE" key...', curses.color_pair(3))
     stdscr.refresh()
-    while 'KEY_BACKSPACE' != stdscr.getkey():
-        pass
+    while True:
+        print(stdscr.getkey())
 
 
-def main(content):
+def main(content, non_stop_mode):
```
Comment (Kamirus) [inline typingtraining]: Here this name does not fit. I'd change to `repeats` or sth like that

File: typingtraining [hunk 2]
```diff
@@ -11,18 +11,22 @@ from lib.tracker import Tracker
...
     def main_inner(stdscr):
         if curses.has_colors():
             # height, width = stdscr.getmaxyx()
             lib = Library()
             # lib.createdefault()
             tracker = Tracker(content(lib))
-            tracker.start_tracking(stdscr)
+            repeats_counter = int(non_stop_mode)
```
Comment (Kamirus) [inline typingtraining]: Consider removing this 'cast', this should be int anyway. Maybe merge this variable with `non_stop_mode`?

**Review by Kamirus — CHANGES_REQUESTED** [review]
Nicely done, thanks for contributing! :1st_place_medal: 
I left some comments that should be at least discussed before approve :smile:

**Review by Kamirus — APPROVED** [review]

File: typingtraining (hunk 2 above)
Comment (insoPL) [reply typingtraining]: While removing cast is possible, merging is not becouse non_stop_mode is variable from Main function which returns main_inner function as return value. Afterwards this main_inner function is send as argument to curses.wrapper. By that time non_stop_mode is const value written inside main_inner, if we merge those variables main_inner would by trying to access non_stop_mode from inside main_inner and fail in doing so.

**Pull request merged**